common/assets/javascripts/analytics.js

```diff
@@ -0,0 +1,8 @@
+function getGaFields (element) {
+  return Array.from(element.attributes).reduce((accumulator, attribute) => {
+    if (/data-ga-field-/.test(attribute.name)) {
+      accumulator[attribute.name.substring(14)] = attribute.value
```

Comment:
Rather than the magic number `14`, you can use a capture group in your regex:
```
const match = attribute.name.match(/^data-ga-field-(.*)/)
if (match) {
  const fieldName = match[1]
  accumulator[fieldName] = attribute.value
}
```